### PR TITLE
style: Change theme to Just The Docs theme

### LINKED ISSUE #5: Change page theme  to Just The Docs theme

### PR DESCRIPTION
**Pull request by lamnpham**
Close #5 
Hi @qndikauri, before anh approve please change page source to GitHub Actions:
Setting and the Page: Under Build and deployment change Source to "GitHub Actions" (We set it at "Deploy from a branch" with master-docs).
Thank You Anh!

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  push:
+    branches: ["main"]
```
Comment (qndikauri) [inline .github/workflows/ci.yml:5]:
```suggestion
    branches: ["master"]
```

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title: Deverlopers
```
Comment (qndikauri) [inline _config.yml:1]:
```suggestion
title: Developers
```

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+description: Dikauri Dev Page
+theme: just-the-docs
+
+url: https://github.com/Dikauri/dikauri.github.io
+
+aux_links:
+  Repositories: https://github.com/lamnpham/lamnpham.github.io
```
Comment (qndikauri) [inline _config.yml:8]: This one should be changed.

File: index.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+title: How To
+layout: default
+nav_order: 1
+has_children: true
+---
+Welcome to our Dev Page
```
Comment (qndikauri) [inline index.md:7]:
```suggestion
Welcome to Dikauri Dev Page
```

**Review by qndikauri — COMMENTED** [review]
Overall LGTM!

File: _config.yml (hunk 3 above)
Comment (lamnpham) [reply _config.yml:8]: I will update when I add the how to pull requests.
Thanks!

**Pull request merged**